### ISSUE TITLE
InputOperand of AttributeRelationship can't be another attribute

### ISSUE THREAD

**Issue by sven-n** [MUnique/OpenMU#86]
Currently, the input operand of a AttributeRelationship can only be a constant numerical value. This is sufficient in probably 99% of the use cases.

Example, where it's not:
There is an excellent option which adds 1 maximum base damage per 20 character levels. Currently, this special case is handled in code - but this is an ugly workaround.

It would be great if we could add an additional (second) input attribute which will be multiplied (or whatever is the input operand) with the currently existing constant numerical value and the other existing input attribute.

Basically like this:
Target element value increases by: ```InputAttribute1 [*|+|^] InputAttribute2  [*|+|^] InputOperand```

**Comment by sven-n** [comment]
After writing my new blog post some days ago, I came to the conclusion that this requirement is actually not needed!
The Level/20-Options are possible with the existing infrastructure with Attribute Relationships.
So, to apply YAGNI here, I'll close it as soon as I replaced the workaround.
